vignettes/articles/theme.Rmd

```diff
@@ -20,11 +26,17 @@ output:
 ---
 ```
 
-This screenshot shows the result of applying the above `theme` to [this example dashboard](https://gist.github.com/cpsievert/04bedbe9af652fab8c24a5e5eee0d721) (subsequent screenshots also use this example, but swap out different `theme`s).
+<!--Carson- can you expand on when you need to specify Bootstrap version- should we recommend always? what is the default if not stated?-->
+
+This screenshot shows the result of applying the above `theme` to [this example dashboard](https://gist.github.com/cpsievert/04bedbe9af652fab8c24a5e5eee0d721) that [uses Shiny](theme.html) (subsequent screenshots also use this example, but swap out different `theme`s).
 
 ![](images/theme-minty.png){.snippet}
 
-In addition to choosing from a handful of [Bootswatch themes](https://bootswatch.com/), you can also accomplish a significant amount of custom theming by changing the [main colors and fonts](https://rstudio.github.io/bslib/articles/theming.html#main-colors-fonts) (e.g., `bg`, `fg`, `primary`, `base_font`, `code_font`, etc.) as well as more specific theming options, [such as `navbar-bg`](https://rstudio.github.io/bslib/articles/bs4-variables.html#navbar-bg). Under the hood, this works by passing the `theme` to `bslib::bs_theme()` which influences the final look of the page by creating a custom build of Bootstrap (and Flexdashboard) CSS.
+## Custom themes
+
+You can also create your own custom theme by setting the [main colors and fonts](https://rstudio.github.io/bslib/articles/theming.html#main-colors-fonts) (e.g., `bg`, `fg`, `primary`, `base_font`, `code_font`, etc.) as well as more specific theming options, [such as `navbar-bg`](https://rstudio.github.io/bslib/articles/bs4-variables.html#navbar-bg). Under the hood, this works by passing the `theme` to `bslib::bs_theme()` which influences the final look of the page by creating a custom build of Bootstrap (and Flexdashboard) CSS.
+
+<!--Carson- do these variables only work for version 4? also, can you combine with a theme- maybe worth saying you can do theme plus variables, or just use variables on their own?-->
```

Comment:
The named arguments of `bs_theme()` work for any version (currently 3 and 4, but we'll also add 5 soon). `?bs_theme` should do a better job of explaining that.

I'm not sure I understand the 2nd question -- all the arguments of `bs_theme()` set new Sass variable defaults, so I don't think the "theme plus variables" concept is useful in this context (it could be in the bootswatch, however)

Reply:
My question was can you layer variables on top of a theme, as in use `cosmo` but lighten the navbar background color, for example.

Reply:
Ahh, I see. In general, mixing Bootswatch with Sass variables is not something I'd like to encourage (Bootswatch themes are not really designed to be themeable), but I'm not entirely against it if you find it useful

Reply:
Got it- I think a lot of folks may just want the theme for the font, then change a handful of variables only (instead of starting from scratch).